src/App.js

```diff
@@ -1,12 +1,13 @@
-import React, { Component } from 'react';
-import './App.css';
-import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import "./App.scss";
+
+import React, { Component } from "react";
+import { BrowserRouter, Route, Switch } from "react-router-dom";
 
 import Home from "./components/home/Home";
-import Customers from "./components/customers/Customers";
+import Customers from "./components/Customers/Customers";
```

Comment:
Are we using capital letters for folders? We should keep it consistent, now some are capital and some not.

Reply:
as discussed - let's keep them lower case. I'll push changes in a sec

Reply:
dun